lib/Validations.php

```diff
@@ -43,6 +43,7 @@
  */
 class Validations
 {
+    private $klass;
```

Comment:
What is this for?

Reply:
https://www.php.net/manual/en/migration82.deprecated.php#migration82.deprecated.core.dynamic-properties

says that dynamic properties are no longer allowable. Declaring:

private $klass

fixes that deprecation error on line 99:
$this->klass = Reflections::instance()->get(get_class($this->model));

Reply:
Ohh, gotcha. Didn't realize there were references to it elsewhere. 👍